README.md

```diff
@@ -173,6 +174,40 @@ class { 'sensu':
 }
 ```
 
+### Installing Plugins
+
+Plugin installation is handled by the `sensu::plugins` class.
```

Comment:
s/installation/management

Installation is a non-idempotent term

Reply:
Change made